packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -387,7 +387,7 @@ class ExpansionPanelList extends StatefulWidget {
   /// 8, 9, 12, 16, 24.
```

Comment:
we should remove the paragraph that says this uses kElevationToShadow, since it doesn't any more

Reply:
Hello @Hixie 
```
  /// Defines elevation for the [ExpansionPanel] while it's expanded.
  ///
  /// By default, the value of elevation is 2.
  final double elevation;
```
This docs is suffecient?

Reply:
Seems reasonable.

Reply:
Thanks @Hixie